test/budou_test.py

```diff
@@ -74,6 +75,43 @@
   {'beginOffset': 9, 'content': u'ご飯'}
 ]
 
+DEFAULT_TOKENS_ZH = [{
+    'dependencyEdge': {'headTokenIndex': 4, 'label': 'VMOD'},
+    'partOfSpeech': {'tag': 'UNKNOWN'},
+    'text': {'beginOffset': 0, 'content': u'随时'}
+  }, {
+    'dependencyEdge': {'headTokenIndex': 2, 'label': 'NN'},
+    'partOfSpeech': {'tag': 'UNKNOWN'},
+    'text': {'beginOffset': 2, 'content': u'互动'}
+  }, {
+    'dependencyEdge': {'headTokenIndex': 0, 'label': 'DOBJ'},
+    'partOfSpeech': {'tag': 'NOUN'},
+    'text': {'beginOffset': 4, 'content': u'交流'}
+  }, {
+    'dependencyEdge': {'headTokenIndex': 4, 'label': 'MARK'},
+    'partOfSpeech': {'tag': 'ADV'},
+    'text': {'beginOffset': 6, 'content': u'并'}
```

Comment:
@luzhang Does it make sense to leave this character alone in Chinese? If it's better to concatenate this word to the former or latter word from lexical perspective, we may want to add a new rule to do that.